stdlib/public/runtime/CMakeLists.txt

```diff
@@ -209,17 +209,21 @@ foreach(sdk ${SWIFT_SDKS})
     if(${SWIFT_SDK_${sdk}_OBJECT_FORMAT} STREQUAL ELF)
       string(TOLOWER "${sdk}" lowercase_sdk)
       set(libpthread -lpthread)
+      set(concurrency_libs)
       set(android_libraries)
       if(${sdk} STREQUAL ANDROID)
         set(android_libraries -llog)
         set(libpthread)
+      elseif(SWIFT_CONCURRENCY_USES_DISPATCH)
+        set(concurrency_libs "-ldispatch -lBlocksRuntime")
       endif()
 
       set(linkfile ${lowercase_sdk}/static-stdlib-args.lnk)
       file(WRITE "${SWIFTSTATICLIB_DIR}/${linkfile}" "
 -ldl
 ${libpthread}
 ${android_libraries}
+${concurrency_libs}
 -lswiftCore
```

Comment:
Hmm, please swap L227 and L226.  IIRC, with Concurrency, swiftCore can make a reference to libdispatch.

Reply:
Done.